src/test/kotlin/org/elm/ide/inspections/ElmIncompletePatternInspectionTest.kt

```diff
@@ -90,6 +90,137 @@ foo it =
             --EOL
 """)
 
+//    fun `test no existing branch in parenthesis`() = checkFixByText("Add missing case branches", """
+//type Foo = Bar | Baz | Qux
+//
+//foo : Foo -> ()
+//foo it =
+//    (<error>case{-caret-}</error> it of<error>)</error>
+//""", """
+//type Foo = Bar | Baz | Qux
+//
+//foo : Foo -> ()
+//foo it =
+//    (case it of
+//        Bar ->
+//            --EOL
+//
+//        Baz ->
+//            --EOL
+//
+//        Qux ->
+//            --EOL
+//    )
+//""")
+
+//    fun `test one existing branch in parenthesis`() = checkFixByText("Add missing case branches", """
+//type Foo = Bar | Baz | Qux
+//
+//foo : Foo -> ()
+//foo it =
+//    (<error>case{-caret-}</error> it of
+//        Baz ->
+//            ()<error>)</error>
+//""", """
+//type Foo = Bar | Baz | Qux
+//
+//foo : Foo -> ()
+//foo it =
+//    (case it of
+//        Baz ->
+//            ()
+//
+//        Bar ->
+//            --EOL
+//
+//        Qux ->
+//            --EOL
+//    )
+//""")
+
+    fun `test one existing branch in parenthesis on new line`() = checkFixByText("Add missing case branches", """
```

Comment:
On `master`, this test fails with this diff:
![image](https://user-images.githubusercontent.com/6695040/103339180-003a7600-4a81-11eb-9359-a34ab4d02206.png)